flexer/runner.py

```diff
@@ -180,6 +180,8 @@ def _get_validation_schema_file(self, event_source, handler):
         schema_file = None
         if handler == 'main.get_resources':
             schema_file = "get_resources.json"
+        elif handler == 'main.get_spend':
```

Comment:
It is safer to check the event_source and not the handler. You can change the resources one too whilst you are on it  :)